src/output.rs

```diff
@@ -212,28 +212,32 @@ fn format_output(ui: &mut Ui, text: &str) {
                     Some(_) | None => {
                         let mut text = RichText::new(span.as_str());
 
-                        text = text.color(ansi_color_to_egui(fg_colour));
+                        if let Some(fg) = fg {
+                            text = text.color(ansi_color_to_egui(fg));
+                        }
 
-                        if bg_colour != Color::Black {
-                            text = text.background_color(ansi_color_to_egui(bg_colour));
+                        if let Some(bg) = bg {
+                            if bg != Color::Black {
+                                text = text.background_color(ansi_color_to_egui(bg));
+                            }
                         }
 
-                        if italic {
+                        if let Some(true) = italic {
                             text = text.italics();
                         }
 
-                        if underline {
+                        if let Some(true) = underline {
                             text = text.underline();
                         }
 
-                        if strikethrough {
+                        if let Some(true) = strikethrough {
                             text = text.strikethrough();
                         }
 
-                        text = match intensity {
-                            cansi::Intensity::Normal => text,
-                            cansi::Intensity::Bold => text.strong(),
-                            cansi::Intensity::Faint => text.weak(),
+                        text = match intensity.unwrap_or(Intensity::Normal) {
+                            Intensity::Normal => text,
+                            Intensity::Bold => text.strong(),
+                            Intensity::Faint => text.weak(),
```

Comment:
Minor change, but I think I slightly prefer this version:
```
text = match intensity {
    Some(Intensity::Bold) => text.strong(),
    Some(Intensity::Faint) => text.weak(),
    Some(Intensity::Normal) | None => text,
};
```

Reply:
That's better. Done.